.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "Hazel/vendor/spdlog"]
 	path = Hazel/vendor/spdlog
 	url = https://github.com/gabime/spdlog
-[submodule "Hazel/vendor/GLFW"]
-	path = Hazel/vendor/GLFW
-	url = https://github.com/TheCherno/glfw
 [submodule "Hazel/vendor/imgui"]
 	path = Hazel/vendor/imgui
 	url = https://github.com/TheCherno/imgui
+[submodule "Hazel/vendor/GLFW"]
+	path = Hazel/vendor/GLFW
+	url = https://github.com/marian-bielcik/glfw
```

Comment:
In my opinion there is no need to move GLFW submodule to another fork.

Reply:
Marian moved the submodules away to an updated version, and made a PR. Until GLFW submodule is solved, it __must__ point away. Before merging it back into the master branch it should point __back__ to the updated submodule.

This is also one of the bullet points of the todo list: `Update submodule in this PR`. Currently we're waiting on the PR in the GLFW module to be merged, before we can let the submodule point back from the fork, to the updated master branch.

Reply:
Update: Cherno did update GLFW and now (after https://github.com/TheCherno/Hazel/pull/82/commits/a406ff85e922b24feb0c5ff06117a4d3b2b48908) this points back to cherno's fork of GLFW.